x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/get_data_streams/index.ts

```diff
@@ -59,13 +61,15 @@ export async function getDataStreams(options: {
   const mappedDataStreams = filteredDataStreams.map((dataStream) => ({
     name: dataStream.name,
     integration: dataStream._meta?.package?.name,
+    // @ts-expect-error
+    lastActivity: dataStream.maximum_timestamp,
```

Comment:
package of `elasticsearch` has not been released yet, this will be removed when the new package is in place

Reply:
Can you update the comment like, remove this error post a date or PR merge. This way its trackable in code as this comment will get lost in time

Reply:
Don't worry, I won't forget. As soon as 8.16 is released I'll get rid of it